invenio_opendata/testsuite/data/cms/cms-csv-files.xml

```diff
@@ -1,4 +1,96 @@
 <collection>
+  <record>
+  <controlfield tag="001">554</controlfield>
+  <datafield tag="024" ind1="7" ind2=" ">
+    <subfield code="2">DOI</subfield>
+    <subfield code="a">10.7483/OPENDATA.CMS.GACK.GEJA</subfield>
+  </datafield>
+  <datafield tag="100" ind1=" " ind2=" ">
+    <subfield code="a">Duarte, Javier</subfield>
+  </datafield>
+  <datafield tag="245" ind1=" " ind2=" ">
+    <subfield code="a">Example CSV output file for SUSYBSMAnalysis-RazorFilter</subfield>
```

Comment:
@jmduarte @katilp Is this the wanted title for the SUSYBSMAnalysis-RazorFilter example output CSV file record?